lua/cfc_entity_stubber/main_loader.lua

```diff
@@ -0,0 +1,79 @@
+cfcEntityStubber = { }
+cfcEntityStubber.stubQueue = { }
+cfcEntityStubber.stubbers = { }
+cfcEntityStubber.oldWeaponStats = { }
+cfcEntityStubber.newWeaponStats = { }
+local stubbersDirectory = "cfc_entity_stubber/stubbers/"
+
+-- MAIN STUBBER FUNCTIONS 
+function cfcEntityStubber.includeStubbers( )
+    local files = file.Find( stubbersDirectory .. "*.lua", "LUA", "datedesc" )
+
+    for _, stubber in ipairs( files ) do
+        MsgC( Color( 41, 41, 41 ), "[", Color( 150, 150, 150 ), "Stubber", Color( 41, 41, 41 ), "] ", Color( 0, 255, 0 ), stubber .. " successfully loaded.\n" )
+        include( stubbersDirectory .. stubber )
+    end
+end
+
+function cfcEntityStubber.getStubs( tab )
+    for _, dir in ipairs( tab ) do
+        local stubFolderPath = "cfc_entity_stubber/" .. dir .. "/"
+        -- List all stubs in the packs folder, sorted by names ascending
+        local stubFiles, stubFolders = file.Find( stubFolderPath .. "*", "LUA", "nameasc" )
+        local emptyFolder = table.IsEmpty( stubFiles )
+        local emptySubFolder = table.IsEmpty( stubFolders )
+
+        if emptyFolder and emptySubFolder then
+            MsgC( Color( 41, 41, 41 ), "[", Color( 150, 150, 150 ), "Stubber", Color( 41, 41, 41 ), "] ", Color( 255, 0, 0 ), "folder " .. dir .. " is empty or doesn't exist.\n" )
+
+            return
+        end
+
+        if not emptySubFolder then
+            for _, stubFile in ipairs( stubFolders ) do
+                local subfolderPath = stubFolderPath .. stubFile
+                local subfolderFiles = file.Find( subfolderPath .. "/*.lua", "LUA", "nameasc" )
+
+                for _, fileName in pairs( subfolderFiles ) do
+                    include( subfolderPath .. "/" .. fileName )
+                end
```

Comment:
this could be a good place to utilize a recursive function.
unless we only want to support files 1 directory deep.
Perhaps that can be left for a later PR revision though

Reply:
Yeah honestly i thought of it too but this *should* be fine for now.